### PR TITLE
LTS 18.0

### PR DESCRIPTION
**Pull request by aschmois**

### CODE REVIEW

**Review by codygman — APPROVED** [review]

File: snapshot.yaml [hunk 1]
```diff
@@ -41,3 +39,9 @@ packages:
   # https://github.com/EdutainmentLIVE/recurly
   - git: https://github.com/EdutainmentLIVE/recurly
     commit: 9c8badd97c662338ed6f1c201df329a3d9e5cd5a
+
+  # mongoDB 2.13.0.2 was not uploaded to hackage for whatever reason
```
Comment (tfausak) [inline snapshot.yaml:43]: Might be worth bugging them about this.

**Review by tfausak — APPROVED** [review]

File: snapshot.yaml (hunk 1 above)
Comment (aschmois) [reply snapshot.yaml:43]: We found out why it wasn't uploaded 😅 it just doesn't work.

Comment (codygman) [reply snapshot.yaml:43]: Is this the reason?

Comment (aschmois) [reply snapshot.yaml:43]: https://github.com/yesodweb/persistent/issues/1284

Comment (tfausak) [reply snapshot.yaml:43]: Is our current game plan still to stick with Persistent 2.12?

Comment (aschmois) [reply snapshot.yaml:43]: Going to see if I can get persistent compiling, otherwise I'm just going to leave the snapshot alone.

**Review by tfausak — APPROVED** [review]

**Pull request merged**